rules/numeric-host/src/lib.rs

```diff
@@ -0,0 +1,138 @@
+use anyhow::{anyhow, Ok};
+use bulwark_wasm_sdk::*;
+use regex::*;
+
+struct NumericHostPlugin;
+
+fn check_numeric_host(req: Request) -> std::result::Result<bool, anyhow::Error> {
+    let host = req
+        .headers()
+        .get("Host")
+        .ok_or(anyhow!("Missing Host Header."))?;
```

Comment:
I think it's probably OK to return an error for this because handler logic should never even get reached if it's missing and it's not the only error that can occur in the function. If it were the only error, it probably wouldn't be worth changing the function signature for something essentially infallible, but since it's not the only one, I think this is fine.